### ISSUE TITLE
Something went wrong with IDs 10, 16, 17, 18: Unexpected token '<', "<!DOCTYPE "... is not valid JSON

### ISSUE THREAD

**Issue by Freebase394** [PrestaShop/PrestaShop#32997]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

![image](https://github.com/PrestaShop/PrestaShop/assets/19652056/9d9de807-00aa-4fc6-ad8c-13ee8c113854)

[errors.csv](https://github.com/PrestaShop/PrestaShop/files/11851380/errors.csv)

### Expected behavior

Something went wrong with IDs 10, 16, 17, 18: Unexpected token '<', "

### Steps to reproduce

1. Go to: Go to Default "Catolog" of default database PrestaShop 8.0.4-
2. Go To Products : Select all the products available....
3. 
![image](https://github.com/PrestaShop/PrestaShop/assets/19652056/dbb1b2d1-07c9-45d8-bcf1-54be32abcb1b)

4.Select Bulk actions --
5. Select "Deactivate Sellection" 
6. And at the end of the process it will give the error : "Something went wrong with IDs 10, 16, 17, 18: Unexpected token '<', ""

### PrestaShop version(s) where the bug happened

8.0.4

### PHP version(s) where the bug happened

7.4.33 (outdated)

### If your bug is related to a module, specify its name and its version

---

### Your company or customer's name goes here (if applicable).

---

**Comment by AureRita** [comment]
Hi @Freebase394 

Thank you for your report, I try to reproduce your issue but as you can see : 

[Untitled_ Jun 26, 2023 5_10 PM.webm](https://github.com/PrestaShop/PrestaShop/assets/100699445/e85d2fde-356f-4d93-bb7c-71d47ee356ba)

I currently can't reproduce it, 

Did I miss something ? 

Waiting for your feedback

**Comment by Freebase394** [comment]
Yes, it seems to have been some anomaly on my end, in my PrestaShop 8.xxx.
Apparently I got this error I don't know how ... maybe I bugged something in the database and started giving this bizarre errors ... I also searched and did not find any references  online... ...
I have however already reinstalled the same version, including more recent versions and also redoing the same procedure and it did not give this error again.

I am sorry for any inconvenience ...

I think we can give the case as closed.
